source/utilities.tex

```diff
@@ -11824,30 +11824,54 @@
 are general-purpose memory resources having the following qualities:
 \begin{itemize}
 \item
-Each resource frees its allocated memory on destruction,
-even if \tcode{deallocate} has not been called for some of the allocated blocks.
+A pool resource has an \defn{upstream memory resource},
+supplied on construction, which is used to allocate memory that
+the pool resource subdivides to satisfy calls to its own
+\tcode{do_allocate} member.
+
+\item
+A \defn{chunk} is a region of storage allocated
+from the upstream memory resource. A pool resource divides
+each chunk into one or more blocks of the same block size.
+Different chunks can have different block sizes, and different
+numbers of blocks.
+
+\item
+A \defn{pool} is a collection of chunks that all have the same
+block size. Each pool resource contains a collection of pools
+with different block sizes.
+
+\item
+On destruction, a pool resource frees all the memory it obtained from
```

Comment:
"On any call to `release()`..."? And then just specify that the destructor always calls `release()`?
Otherwise we'll have either that `release()` is underspecified, or the specification of `release()` duplicates all this excellent wording.